frontend/src/scenes/early-access-features/EarlyAccessFeature.tsx

```diff
@@ -140,9 +140,19 @@ export function EarlyAccessFeature({ id }: { id?: string } = {}): JSX.Element {
                             info={<>A feature flag will be generated from feature name if not provided</>}
                         >
                             {({ value, onChange }) => (
-                                <div>
+                                <Row>
```

Comment:
If we could never use Col or Row or any `antd` stuff again that would be _greaaaaatt_ 

`<Col>` is basically nothing more than `<div>` and `<Row>` nothing more than `<div className="flex">` 

Would help us get closer to finally removing that lib 🙌